lib/rails/generators/channel/templates/channel.coffee.erb

```diff
@@ -0,0 +1,3 @@
+App.<%= channel_name %> = App.cable.subscriptions.create "<%= channel_class %>",
```

Comment:
Yeah, probably fine for now. Gotta be `@App`, though, as it needs to be
exported.

On Thu, Jul 9, 2015 at 11:41 AM, Abdelkader Boudih <notifications@github.com
> wrote:

> In lib/rails/generators/channel/templates/channel.coffee.erb
> <https://github.com/rails/actioncable/pull/17#discussion_r34238812>:
>
> > @@ -0,0 +1,3 @@
> > +App.<%= channel_name %> = App.cable.subscriptions.create "<%= channel_class %>",
>
> What about App ||= {} for now ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/actioncable/pull/17/files#r34238812>.
>